home/utils.py

```diff
@@ -623,6 +625,104 @@ def export_csv_content(queryset: PageQuerySet, response: HttpResponse) -> None:
         writer.writerow(row)
 
 
+def get_serialized_ordered_set(
```

Comment:
I think we should consider pulling the imports and exports for each thing into a different place? the utils file is become really large and cumbersome